src/chopper_count.cpp

```diff
@@ -15,29 +16,50 @@ void initialize_argument_parser(seqan3::argument_parser & parser, chopper::count
+    parser.add_option(config.data_file,
+                      'f', "data_file",
+                      "Give me a filename to a seqinfo file.",
                       seqan3::option_spec::required);
-    parser.add_option(config.hll_dir, 'd', "hll-dir",
-                      "A directory path where the HyperLogLog sketches will be stored in files. "
-                      "When this is not given, the sketches will not be stored in files.",
-                      seqan3::option_spec::standard,
-                      seqan3::output_directory_validator{});
-    parser.add_option(config.column_index_to_cluster, 'c', "column-index", "The column index by which to cluster.");
-    parser.add_option(config.num_threads, 't', "threads", "Number of threads.");
-    parser.add_option(config.k, 'k', "kmer-size", "The kmer size to count minimisers.");
-    parser.add_option(config.w, 'w', "window-size", "The window size for minimisers.");
-    parser.add_option(config.sketch_bits, 's', "sketch-bits",
+
+    parser.add_option(config.output_prefix,
+                      'o', "output-prefix",
+                      "chopper count creates a `[PREFIX].weights` file that contains an estimated weight for each user "
+                      "bin which is used in chopper layout. Additionally a directory [PREFIX].sketches is created and "
+                      "one `.sketch` file per user bin is computed. The sketch files improve the layout further.",
+                      seqan3::option_spec::required);
+
+    parser.add_option(config.column_index_to_cluster,
+                      'c', "column-index",
+                      "The column index by which to cluster. Clustering is not supported at the moment",
+                      seqan3::option_spec::hidden);
+
+    parser.add_option(config.num_threads,
+                      't', "threads",
+                      "Number of threads.");
+
+    parser.add_option(config.k,
+                      'k', "kmer-size",
+                      "The kmer size to count minimisers.");
+
+    parser.add_option(config.w,
+                      'w', "window-size",
+                      "The window size for minimisers.");
+
+    parser.add_option(config.sketch_bits,
+                      's', "sketch-bits",
                       "The number of bits the HyperLogLog sketch should use to distribute the values into bins.",
                       seqan3::option_spec::standard,
                       seqan3::arithmetic_range_validator{5, 32});
+
+    parser.add_flag(config.disable_sketches_output, '\0', "dont-output-sketches",
+                    "Although the sketches will improve the layout you might want to disable this for saving disk "
+                    "space by not writing out sketch files. Note that you cannot use the options --estimate-unions nor "
+                    "--rearrange-bins in chopper layout then.");
```

Comment:
Same as `output_prefix`, begin with stating what it does and then what the consequence is.

Maybe you can also mention that the sketches are still computed - but not written - when this flag is enabled.